Fix issue 29587: remove unhelpful link to Unicode (#29713)
mdn/content: files/en-us/glossary/internationalization/index.md

---

> title: Internationalization

Reviewer: What do you think about adding `i18n` into the title so we have site search hits for it?

```suggestion
title: Internationalization (i18n)
```

Author: Yeah, that's a good idea.

slug: Glossary/Internationalization
page-type: glossary-definition
---

{{GlossarySidebar}}

**Internationalization**, often shortened to "i18n", is the practice of designing a system in such a way that it can easily be adapted for different target audiences, that may vary in region, language, or culture.

The complementary process of adapting a system for a specific target audience is called {{glossary("Localization")}}.

> Among other things, internationalization requires support for:
> 
> - different writing systems
> - different units of measure (currency, °C/°F, km/miles, etc.)
> - different time and date formats
> - different keyboard layouts

Reviewer: What do you think?

```suggestion
Among other things, internationalization covers adapting to differences in:

- writing systems
- units of measure (currency, °C/°F, km/miles, etc.)
- time and date formats
- keyboard layouts
```
